modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/js/ApplicationsMenu.js

```diff
@@ -346,6 +351,28 @@ const ApplicationsMenu = ({
 		onClose: () => setVisible(false),
 	});
 
+	useEventListener(
+		'keydown',
+		(event) => {
+			const isCMDPressed = event.metaKey || event.ctrlKey;
```

Comment:
Is it possible this means that `CTRL + SHIFT + M` also works on Mac?

Reply:
Yes. CTRL + SHIFT + M and COMMAND + SHIFT + M.

Reply:
Need I detect the OS and don't allow this CTRL option on Mac for just using COMMAND?

Reply:
I think so... I don't think CTRL is expected in OSX...

Reply:
Fixed! 3b8c9e7